### PR TITLE
create capstone project PR#1

### PR DESCRIPTION
**Pull request by efmomoh**
In this project, I did the following:
- [ ] Setup mobile about and home pages
- [ ] Use media query for the desktop version
- [ ] Use HTML, CSS, and javascript to layout the page
- [ ] adds readme file

NOTE: I HAVE BEEN FACING HARDWARE ISSUES WITH MY COMPUTER RECENTLY WHICH CAUSED ME TO LOSE MY  MOBILE SETUP CAPSTONE PROJECT LOCALLY. ALSO, LINTERS TESTS HAVE NOT BEEN RUNNING ON MY COMPUTER DUE TO THE SAME HARDWARE ISSUES AND IT TOOK  ABOUT TWO DAYS WHICH HAS RESULTED IN DELAY AND LATENESS IN MAKING THIS PULL REQUEST. KINDLY TAKE CONSIDERATION WHEN MAKING A REVIEW OF THIS PROJECT, THANK YOU!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,139 @@
...
+
+If you like this project please give it a star and follow me on GitHub LinkedIn and Twitter
+, you are also welcome to any suggestions🙏
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my Microverse studying partners, morning session, standup team and the entire team at Microverse for aiding and encouraging me and for making this capstone project a success.
+
```
Comment (V-Blaze) [inline README.md]: > The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.

- You have done fantastic so far :+1: :100: You did great building out your capstone, but there are a few changes you need to make before moving to the next milestone. :+1: :+1: 
- [ ] As stated in the project requirement, you need to give credit to the original creator of the design. I'll suggest you add this to the  `Acknowledgments section` of your readme file. :+1: :+1:

File: README.md [hunk 2]
```diff
@@ -1,2 +1,139 @@
...
+### Run tests
+
+```sh
+   1. npx hint.
+   2. npx stylelint "**/*.{css,scss}"
+```
+## 🚀 Live Demo <a name="live-demo"></a>
+
+ - [Live Demo Link](https://efmomoh.github.io/Capstone-Project.git/)
```
Comment (V-Blaze) [inline README.md]: - [ ] The link to the live demo of your project leads to a 404 error page, I'll suggest you correctly deploy your project with Githuibpages and add the correct link here. :+1: :+1: 

- - Screenshot:
Screenshot from 2023-08-03 10-02-34

File: html/home.html [hunk 3]
```diff
@@ -236,6 +235,7 @@ <h6 class="h6 text-wrap ft-h6 fs-4"> &copy; Creative Commons Liberia</h6>
             </div>
```
Comment (V-Blaze) [inline html/home.html:229]: - [ ] When I am on the `home page` and click on the About page on the nav bar, it doesn't lead to the About page rather it stays on the home page. same when I click on the Home link while on the `About page`, it doesn't navigate, I'll suggest you fix this. :+1: :smile: 

- - Screenshot:
Selection_008

- [ ] You did great implementing the mobile nav bar, but looking at the given design, the LOGO is not visible on mobile screens, I'll suggest you remove that to match the given design:

- - Screenshot:
 
Selection_009

File: html/home.html [hunk 4]
```diff
@@ -236,6 +235,7 @@ <h6 class="h6 text-wrap ft-h6 fs-4"> &copy; Creative Commons Liberia</h6>
...
         </div>
```
Comment (V-Blaze) [inline html/home.html:230]: > All page elements have similar spacing, borders, shadows, and paddings than the orignal design.

- [ ] Looking at the given design, you can see that when a program card is hovered, there is a white border around the card, this is not currently implemented in your program cards, so I'll suggest you do so. :+1: :+1: 

- - Screenshot reference:
Screenshot from 2023-08-03 10-12-13

File: html/home.html [hunk 5]
```diff
@@ -212,7 +212,6 @@ <h5 class="h5 text-center">Featured Speakers</h5>
...
         <!-- <button type="submit" class="moreBtn text-center">MORE<i class="bi bi-chevron-down"></i></button> -->
     </secttion>
-    
     <!-- address and footer section  -->
```
Comment (V-Blaze) [inline html/home.html]: - [ ] You did great generating the speaker section dynamically, :+1: :+1:  But looking at the design, you can see that when on a mobile screen, the list of speakers is just one row, you seem to have two on each row, I'll suggest you adjust yours to look like the provided design. :+1: :+1: :smile: 

- - Your design:
 
Screenshot from 2023-08-03 10-16-31


- - Given design:
Screenshot from 2023-08-03 10-16-14

File: html/home.html [hunk 6]
```diff
@@ -212,7 +212,6 @@ <h5 class="h5 text-center">Featured Speakers</h5>
...
     <address class="container-fluid address">
```
Comment (V-Blaze) [inline html/home.html:210]: - [ ] All the `img` tag in your code seems to be missing an `alt` attribute. I'll suggest you add the salt attribute to all your images as they are used by screen readers to know what the image is all about. :smile: :wave: :+1:

File: html/home.html [hunk 7]
```diff
@@ -212,7 +212,6 @@ <h5 class="h5 text-center">Featured Speakers</h5>
         </div>
         <!-- <button type="submit" class="moreBtn text-center">MORE<i class="bi bi-chevron-down"></i></button> -->
```
Comment (V-Blaze) [inline html/home.html]: - [ ] It is against the best practices to leave unused code as a comment on your code base, I'll suggest you remove this as it makes the code base look unprofessional. :small_airplane: :smile:

File: html/home.html [hunk 8]
```diff
@@ -236,6 +235,7 @@ <h6 class="h6 text-wrap ft-h6 fs-4"> &copy; Creative Commons Liberia</h6>
             </div>
         </div>
     </footer>
+    
     <script src="/javaScripts/index.js"></script>
```
Comment (V-Blaze) [inline html/home.html:233]: - [OPTIONAL] `createSpeakers();` I'll suggest you load the feature speakers in the featured speakers container when the window is completely loaded. This prevents any potential issues with incomplete loading, such as missing images or other content, and guarantees a smoother user experience. You can achieve this by passing the function as a call back like this. `window.addEventListener("DOMContentLoaded", () => createSpeakers());` :smile: :small_airplane: :wave:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @efmomoh 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- UI looks great✔
-  Clean and well-documented code 👋 
-  Correct use of semantic tags✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 9]
```diff
@@ -1,2 +1,139 @@
...
+
+### Run tests
+
+```sh
+   1. npx hint.
+   2. npx stylelint "**/*.{css,scss}"
+```
+## 🚀 Live Demo <a name="live-demo"></a>
+
+ - [Live Demo Link](https://github.com/efmomoh/Capstone-Project/deployments/github-pages)
```
Comment (Alusp) [inline README.md:97]: - [ ] Great job in implementing the changes requested by the previous reviewer, :clap: However, as requested by the previous reviewer, your Live demo link is broken and showing 404 page,  Please resolve it. :slightly_smiling_face:

File: README.md [hunk 10]
```diff
@@ -1,2 +1,139 @@
...
+- Add a chatbot using API
+- Adds Home and About page section
+- Adds mobile menu toogler
+- Desktop version
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 contributing <a name="contributing"></a>
+
+You are welcome, and click <a href="https://github.com/efmomoh/Capstone-Project.git/issues">here</a> for contribution, issues, and feature request 🙏
```
Comment (Alusp) [inline README.md]: - [OPTIONAL] Please ensure that the link to your issues page point correctly as it is currently showing 404 page, possible solution would be replacing the href link to `https://github.com/efmomoh/Capstone-Project/issues`  :slightly_smiling_face: 

Issues 404

File: README.md [hunk 11]
```diff
@@ -1,2 +1,139 @@
...
+
+If you like this project please give it a star and follow me on GitHub LinkedIn and Twitter
+, you are also welcome to any suggestions🙏
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my Microverse studying partners, morning session, standup team and the entire team at Microverse for aiding and encouraging me and for making this capstone project a success.
+
```
Comment (Alusp) [inline README.md]: - [ ] As mentioned by the previous reviewer, please ensure that you acknowledge and give credit to the Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07) , acknowledging the original designer in the README file is a simple but meaningful way to show appreciation, comply with licensing requirements, and foster a collaborative and respectful environment within the open-source community. :slightly_smiling_face:

File: css/home.css
Comment (Alusp) [inline css/home.css:1]: ` .campaign {`
   ` color: #ec4327;`
    `border: 2px solid #ec4327;`
    `font-size: 16px;`
    `padding: 0.2px;`
    `letter-spacing: -0.03em;`
    `width: auto;`
  `}`

- [OPTIONAL] Please ensure that you button design matches the given design, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. Please resolve it. :slightly_smiling_face: 

Below is an illustrations of yours and the given footer Design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

Student Button


</td>
      <td>

Design Button

</td>
    </tr>
  </table>

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi  @efmomoh, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 close

 ## Highlights

- [ ] Linters set-up ✔️
- [ ] Commit messages descrptive ✔️
- [ ] Professional Github Flow used ✔️
- [ ] Professional PR Summary Flow used ✔️
- [ ] UI looks good :heavy_check_mark: 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by efmomoh** [comment]
THE DEPOLY PAGE HAS NO CONTENTS FOR NOW EXCEPT THE README FILE

**Comment by efmomoh** [comment]
I WORKED ONLY ON THIS BRANCH THAT CONTAINS THE PULL REQUEST

**Comment by sumairq** [comment]
You should rename your home.html file to index.html and move it to the root directory and that should solve your problem.

**Comment by efmomoh** [comment]
> You should rename your home.html file to index.html and move it to the root directory and that should solve your problem.

ok, thanks!

File: README.md [hunk 12]
```diff
@@ -1,2 +1,141 @@
...
+```sh
+   1. npx hint.
+   2. npx stylelint "**/*.{css,scss}"
+```
+## 🚀 Live Demo <a name="live-demo"></a>
+
+ - [Live Demo Link](https://efmomoh.github.io/Capstone-Project/)
+
+
```
Comment (sumairq) [inline README.md]: - [ ] Live demo shows the Readme file.  

  Here is what you can do to fix this problem 
- Rename the home.html to index.html 
- Move index.html to the root/main directory of the project.

**Review by sumairq — CHANGES_REQUESTED** [review]
## Hi @efmomoh , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 12 above)
Comment (efmomoh) [reply README.md]: So how do I make the pages to like? please help

Comment (sumairq) [reply README.md]: If you commit the changes and push the commits the deployment should be retriggered automatically.

**Comment by efmomoh** [comment]
It is all ready

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @efmomoh, 
I'm Andres.

There is still room for improvement but you put enough effort into this project. It is approved and if you want to make it better here is what you can do:    🚀 

- [OPTIONAL] Consider reviewing the [`GitHub pages`](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site) documentation, and update the deployment of your project accordingly. Currently, I still can't view the live version of your portfolio! 🤓 🔧 




## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**